src/FixedDataTable.js

```diff
@@ -818,11 +818,11 @@ class FixedDataTable extends React.Component {
 
   _renderRows = (/*number*/ offsetTop, fixedCellTemplates, fixedRightCellTemplates, scrollableCellTemplates,
     bodyHeight) /*object*/ => {
-    const { scrollEnabledY, scrolling } = scrollbarsVisible(this.props);
+    const { scrollEnabledY } = scrollbarsVisible(this.props);
     const props = this.props;
     return (
       <FixedDataTableBufferedRows
-        isScrolling={scrolling}
+        isScrolling={props.scrolling}
```

Comment:
dammit... This change came here, but since it's already pushed to beta, it shouldn't be harmful